.pre-commit-config.yaml

```diff
@@ -37,5 +37,13 @@ repos:
           - --ini
           - .bandit
 
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v1.8.0
```

Comment:
Please use SHA instead of tag

Reply:
Done: commit 74ef126b5aeee3c10c6b9d591abc5f7d9fe32f26